backends/cxxrtl/cxxrtl_backend.cc

```diff
@@ -628,6 +628,22 @@ std::string escape_cxx_string(const std::string &input)
 	return output;
 }
 
+std::string basename(const std::string &filepath)
+{
+#ifndef _WIN32
+	const std::string dir_seps = "/";
+#else
+	const std::string dir_seps = "\\/";
+#endif
+	size_t sep_pos = filepath.find_last_of(dir_seps);
+	if (sep_pos != std::string::npos) {
```

Comment:
Formatting nit: no curly braces when there's a single statement, also for the else clause below

Reply:
Updated